common/src/main/java/tachyon/worker/WorkerClient.java

```diff
@@ -99,13 +99,17 @@ public WorkerClient(NetAddress workerNetAddress, ExecutorService executorService
    * Updates the latest block access time on the worker.
    *
    * @param blockId The id of the block
-   * @throws IOException
+   * @throws BlockDoesNotExistException if the blockId is not found
+   * @throws IOException if an I/O error occurs
    */
-  public synchronized void accessBlock(long blockId) throws IOException {
+  public synchronized void accessBlock(long blockId) throws IOException,
+          BlockDoesNotExistException {
     mustConnect();
 
     try {
       mClient.accessBlock(blockId);
+    } catch (BlockDoesNotExistException e) {
```

Comment:
Do we need these catch clauses if we are just rethrowing the exception? This applies to multiple locations in the WorkerClient class.